get-started.md

```diff
@@ -0,0 +1,23 @@
+# How to get started in the Grafana Labs bug bounty program
+
+* **Grafana OSS** - we offer docker images that are synced with the `main` branch. This makes it easy to get started right away and be sure that you're looking for issues in an untouched instance. To spin up Grafana OSS in docker, run the following command:
+
+    `docker run -p 3000:3000 grafana/grafana-oss:main` and then visit http://localhost:3000 and login with `admin:admin`
+
+* **Mimir** - you can get started by following [Play with Grafana Mimir](https://grafana.com/tutorials/play-with-grafana-mimir/).
+
+## Understanding the threat model for Grafana
+
+Please refer to the following [documentation page](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/) in order to understand the roles and permissions in Grafana.
+
+Installed non-core plugins are considered to be trusted so any vulnerability affecting those will be handled outside of the bug bounty program. 
```

Comment:
Could be confusing: trusted but outside of the bug bounty program

Reply:
True, how about phrasing it like this: 
```suggestion
Non-core plugins are considered out of scope as Grafana administrators install them at their own risk. 
```

Reply:
That's crystal clear for me now, thank wou